ui-components/scripts/controllers/products.controller.js

```diff
@@ -5,18 +5,19 @@
 
   app.controller("ProductsController", ProductsController);
 
-  ProductsController.$inject = ['$scope', 'Api', 'users', '$uibModal', '$cacheFactory', 'localStorageService'];
+  ProductsController.$inject = ['$scope', 'Api', '$rootScope', '$uibModal', '$cacheFactory', 'localStorageService'];
 
-  function ProductsController($scope, Api, users, $uibModal, $cacheFactory, localStorageService){
+  function ProductsController($scope, Api, $rootScope, $uibModal, $cacheFactory, localStorageService){
 
     var cache = $cacheFactory.get('productsCache'),
-        selectedKey = 'selectedItem';
+        selectedKey = 'selectedItem',
+        token = $rootScope.token;
```

Comment:
Do you think this `token`-thing products related? It looks to me like a common thing, I don't think this controller is the best place to have it